app/services/badges/award_community_wellness.rb

```diff
@@ -0,0 +1,67 @@
+module Badges
+  class AwardCommunityWellness
+    REWARD_STREAK_WEEKS = [1, 2, 4, 8, 16, 32].freeze
+
+    def self.call
+      new.call
```

Comment:
I started off from the `def self.call(...)` style and never noticed this when cleaning up. I was aiming for consistency but it's not exactly that use case here since we don't benefit from the initializer and there are other services that are implemented directly in the static method.

Conclusion: Great input here, I'm pushing the changes in a bit. Thanks!